cypress.json

```diff
@@ -1 +1,3 @@
-{}
```

Comment:
It's odd that you had to add this to get your tests to pass. I wonder how this will affect us in the end.